### PR TITLE
Ref Guide - CLM

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/10589

### CODE REVIEW

File: modules/administration/pages/content-lifecycle.adoc [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-Content Lifecycle Management allows you to customize and test packages before updating production systems.
+Content lifecycle management allows you to customize and test packages before updating production clients.
 This is especially useful if you need to apply updates during a limited maintenance window.
 
-Content Lifecycle Management allows you to select software channels as sources, adjust them as required for your environment, and thoroughly test them before installing onto your production systems.
+Content lifecycle management allows you to select software channels as sources, adjust them as required for your environment, and thoroughly test them before installing onto your production clients.
 
 While you cannot directly modify vendor channels, you can clone them and then modify the clones by adding or removing packages and custom patches.
-You can then assign these cloned channels to test systems to ensure they work as expected.
+You can then assign these cloned channels to test clients to ensure they work as expected.
```
Comment (keichwa) [inline modules/administration/pages/content-lifecycle.adoc]:
```suggestion
You can assign these cloned channels to test clients to ensure they work as expected.
```

File: modules/administration/pages/content-lifecycle.adoc [hunk 2]
```diff
@@ -129,14 +130,14 @@ When the project has been built, the built sources can be sequentially promoted
 
 
 
-== Assign Systems to Environments
+== Assign Clients to Environments
 
 When you build and promote content lifecycle projects, it creates a tree of software channels.
```
Comment (keichwa) [inline modules/administration/pages/content-lifecycle.adoc]: who is "it"?

File: modules/reference/pages/clm/clm-filters.adoc [hunk 3]
```diff
@@ -1,15 +1,5 @@
 = Filters
 
-.Procedure: Add a new Filter to a Content Lifecycle Project
+In the menu:Content Lifecycle[Filters] section, you can create various types of filters to control the content that is used when a content lifecycle project is built.
```
Comment (keichwa) [inline modules/reference/pages/clm/clm-filters.adoc]: sentence too long ;)

```suggestion
In the menu:Content Lifecycle[Filters] section, you can create various types of filters.
With the filters you control the content that is used when a content lifecycle project is built.
```

File: modules/administration/pages/content-lifecycle.adoc (hunk 2 above)
Comment (Loquacity) [reply modules/administration/pages/content-lifecycle.adoc]: Oof, {productname}? Maybe?

Comment (keichwa) [reply modules/administration/pages/content-lifecycle.adoc]: yes, sounds better.  Or "the power".

**Pull request merged**